src/Module/Calendar/Event/Form.php

```diff
@@ -0,0 +1,253 @@
+				   || $orig_event['allow_gid']
+				   || $orig_event['deny_cid']
+				   || $orig_event['deny_gid']) {
+			$share_checked = ' checked="checked" ';
+		}
+
+		// In case of an error the browser is redirected back here, with these parameters filled in with the previous values
+		if (!empty($request['nofinish'])) {
+			$orig_event['nofinish'] = $request['nofinish'];
+		}
+		if (!empty($request['summary'])) {
+			$orig_event['summary'] = $request['summary'];
+		}
+		if (!empty($request['desc'])) {
+			$orig_event['desc'] = $request['desc'];
+		}
+		if (!empty($request['location'])) {
+			$orig_event['location'] = $request['location'];
+		}
+		if (!empty($request['start'])) {
+			$orig_event['start'] = $request['start'];
+		}
+		if (!empty($request['finish'])) {
+			$orig_event['finish'] = $request['finish'];
+		}
+
+		$n_checked = (!empty($orig_event['nofinish']) ? ' checked="checked" ' : '');
+
+		$t_orig = $orig_event['summary'] ?? '';
+		$d_orig = $orig_event['desc'] ?? '';
+		$l_orig = $orig_event['location'] ?? '';
+		$eid    = $orig_event['id'] ?? 0;
+		$cid    = $orig_event['cid'] ?? 0;
+		$uri    = $orig_event['uri'] ?? '';
+
+		if ($cid || $mode === 'edit') {
+			$share_disabled = 'disabled="disabled"';
+		}
+
+		$sdt = $orig_event['start'] ?? 'now';
+		$fdt = $orig_event['finish'] ?? 'now';
+
+		$syear  = DateTimeFormat::local($sdt, 'Y');
+		$smonth = DateTimeFormat::local($sdt, 'm');
+		$sday   = DateTimeFormat::local($sdt, 'd');
+
+		$shour   = !empty($orig_event) ? DateTimeFormat::local($sdt, 'H') : '00';
+		$sminute = !empty($orig_event) ? DateTimeFormat::local($sdt, 'i') : '00';
+
+		$fyear  = DateTimeFormat::local($fdt, 'Y');
+		$fmonth = DateTimeFormat::local($fdt, 'm');
+		$fday   = DateTimeFormat::local($fdt, 'd');
+
+		$fhour   = !empty($orig_event) ? DateTimeFormat::local($fdt, 'H') : '00';
+		$fminute = !empty($orig_event) ? DateTimeFormat::local($fdt, 'i') : '00';
+
+		if (!$cid && in_array($mode, [self::MODE_NEW, self::MODE_COPY])) {
+			$acl = ACL::getFullSelectorHTML($this->page, $this->session->getLocalUserId(), false, ACL::getDefaultUserPermissions($orig_event));
+		} else {
+			$acl = '';
+		}
+
+		// If we copy an old event, we need to remove the ID and URI
+		// from the original event.
+		if ($mode === self::MODE_COPY) {
+			$eid = 0;
+			$uri = '';
+		}
+
+		$this->page['aside'] .= CalendarExport::getHTML($this->session->getLocalUserId());
+
+		$tpl = Renderer::getMarkupTemplate('calendar/event_form.tpl');
+
+		return Renderer::replaceMacros($tpl, [
+			'$post' => $this->baseUrl . '/calendar/api/create',
+			'$eid'  => $eid,
+			'$cid'  => $cid,
+			'$uri'  => $uri,
+
+			'$title'       => $this->t('Event details'),
+			'$desc'        => $this->t('Starting date and Title are required.'),
+			'$s_text'      => $this->t('Event Starts:') . ' <span class="required" title="' . $this->t('Required') . '">*</span>',
+			'$s_dsel'      => Temporal::getDateTimeField(
```

Comment:
Expected PHP CS barking.
```suggestion
			'$title'  => $this->t('Event details'),
			'$desc'   => $this->t('Starting date and Title are required.'),
			'$s_text' => $this->t('Event Starts:') . ' <span class="required" title="' . $this->t('Required') . '">*</span>',
			'$s_dsel' => Temporal::getDateTimeField(
```